tao/.htaccess

```diff
@@ -2,6 +2,6 @@ Header set Access-Control-Allow-Origin *
 Options +FollowSymLinks
 RewriteEngine on
 
-SetEnvIf Request_URI ^.*$ ROOT_URL=tourism.documentation.linkalab-cloud.com/tao
+SetEnvIf Request_URI ^.*$ ROOT_URL=http://tourism.documentation.linkalab-cloud.com/tao
```

Comment:
Is it possible to use https? Otherwise with w3id it will go through https -> http, which has problems in some applications

Reply:
We don't have https in place right now but would soon move the files to GitHub